src/chat/buttons.js

```diff
@@ -0,0 +1,27 @@
+import { Message } from "../config/message.js";
+import Config from "../config/config.js";
+import { clickEvent } from "../events/clickevent.js";
+
+export default class Buttons {
+  #config;
+  #chatComponent;
+  #timer;
+  constructor(config, chatComponent, timer, pokeButtonDOM, endDayButtonDOM) {
+    this.#config = config;
+    this.#chatComponent = chatComponent;
+    this.#timer = timer;
+    this.#addEventListener(pokeButtonDOM, endDayButtonDOM);
+  }
+
+  #addEventListener(pokeButtonDOM, endDayButtonDOM) {
```

Comment:
FYI: This is very bad design.
The class name is `Buttons`. Very generic. And you tied this class very closely with the types of buttons you have.

---

Just take note first. For now, I'll aim for completion first